### PR TITLE
module 3: Components

### PR DESCRIPTION
**Pull request by d1rtych**

### CODE REVIEW

File: src/App.css
Comment (valentynkinakh) [inline src/App.css:1]: **nitpick:** If this file is empty maybe we can remove it?

File: src/App.tsx [hunk 1]
```diff
@@ -27,29 +42,44 @@ const App: React.FC = () => {
     console.log(query);
   }
 
+  const handleSortChange = (newSort: string) => {
+    setSelectedSort(newSort);
+    console.log(`Sorting movies by: ${newSort}`);
```
Comment (valentynkinakh) [inline src/App.tsx]: **nitpick:** `console.log` is redundant here

File: src/App.tsx [hunk 2]
```diff
@@ -27,29 +42,44 @@ const App: React.FC = () => {
...
+  };
```
Comment (valentynkinakh) [inline src/App.tsx]: **suggestion:** As we pass this function as a prop, it's better to wrap it in `useCallback`

File: src/components/MovieDetails/MovieDetails.test.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import { mockMovie } from '../../utils/mocks/movie.mock';
+
+jest.mock('../../utils/utils', () => ({
+  formatRuntime: jest.fn().mockReturnValueOnce('2h 22min'),
+}));
+
+describe('MovieDetails', () => {
+  test('renders movie details when movie is selected', () => {
+    render(
+      <MovieContext.Provider value={{ selectedMovie: mockMovie, setSelectedMovie: () => {} }}>
```
Comment (valentynkinakh) [inline src/components/MovieDetails/MovieDetails.test.tsx]: **suggestion:** I'd personally suggest to create helper function for providers(e. g. `createWrapper()`) as in the future we can have multiple contexts

File: src/App.tsx (hunk 1 above)
Comment (d1rtych) [reply src/App.tsx]: Removed

File: src/App.tsx (hunk 2 above)
Comment (d1rtych) [reply src/App.tsx]: done

File: src/components/MovieDetails/MovieDetails.test.tsx (hunk 3 above)
Comment (d1rtych) [reply src/components/MovieDetails/MovieDetails.test.tsx]: done

**Pull request merged**